src/components/Dashboard/Dashboard.tsx

```diff
@@ -1,8 +1,76 @@
-import React, { ReactElement } from 'react'
+import React, {ReactElement} from 'react'
 import styles from './Dashboard.module.css'
+import CanvasDraw from "react-canvas-draw";
+import malaria from './malaria.png';
+import {ActionToolbar} from "./ActionToolbar/ActionToolbar";
+import {ImageToolbar} from "./ImageToolbar/ImageToolbar";
 
-export const Dashboard = (): ReactElement => (
-  <div className={styles.container}>
-    <p>I am the Dashboard - please implement me</p>
-  </div>
-)
+
+export const Dashboard = (): ReactElement => {
+  let canvas: CanvasDraw | null;
+
+  //TODO: Are pictures of a fixed size? we might need to adjust the canvas when the picture size changes
+  //TODO: Canvas is working fine but zoom in and zoom out miss don't keep image at center, workable but not great UX.
+  const saveAction = () => {
+    if (canvas) {
+      localStorage.setItem(
+          "savedDrawing",
+          canvas.getSaveData()
+      )
+    }
+  }
+
+  const undoAction = () => {
+    if (canvas) {
+      canvas.undo()
+    }
+  }
+
+  const editAction = () => {
+    alert('Not implemented')
+  }
```

Comment:
I'm not really a fan of including stuff which is not implemented yet - they tend to get in the way of refactoring and makes life more complex than needed. I think we should remove them and create a story with a vertical slice for these items - YAGNI